src/main/java/org/amahi/anywhere/activity/NavigationActivity.java

```diff
@@ -78,6 +93,25 @@ protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_navigation);
 
+		if(isATV()){
+			Boolean isFirstRun = getSharedPreferences("PREFERENCE", MODE_PRIVATE).getBoolean("isFirstRun", true);
```

Comment:
There's a separate class [util/Preferences.java](https://github.com/amahi/android/blob/master/src/main/java/org/amahi/anywhere/util/Preferences.java#L55) in our project. We could use that for accessing shared preferences instead of polluting the code here.